src/main/java/by/itacademy/javaenterprise/lepnikau/app/dao/impl/PersonDAOImpl.java

```diff
@@ -0,0 +1,53 @@
+package by.itacademy.javaenterprise.lepnikau.app.dao.impl;
+
+import by.itacademy.javaenterprise.lepnikau.app.dao.PersonDAO;
+import by.itacademy.javaenterprise.lepnikau.app.entity.Person;
+import by.itacademy.javaenterprise.lepnikau.app.hibernate.HibernateUtil;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.persistence.EntityManager;
+
+public class PersonDAOImpl implements PersonDAO {
+
+    private static final Logger logger = LogManager.getLogger(PersonDAOImpl.class);
+
+    private EntityManager eManager;
+
+    public PersonDAOImpl(EntityManager eManager) {
+        this.eManager = eManager;
+    }
+
+    @Override
+    public Person save(Person person) {
+        try {
+            eManager.getTransaction().begin();
+            eManager.persist(person);
+            eManager.getTransaction().commit();
+            return person;
+        } catch (Exception e) {
+            logger.error(e.getMessage());
```

Comment:
тут ты теряешь stacktrace, это ошибка.
```suggestion
            logger.error(e.getMessage(), e);
```